gulpfile.js

```diff
@@ -286,13 +286,13 @@
 			const CLI_VERSION = require(path.join(CLI_PATH, "package.json")).version;
 			const CONTROLS_PATH = path.join(__dirname, "node_modules", "@chartiq", "finsemble-react-controls");
 			const CONTROLS_VERSION = require(path.join(CONTROLS_PATH, "package.json")).version;
-			
-			// Check e2o version
+
+			// Check electron version
 			const E2O_PATH = path.join(__dirname, "node_modules", "@chartiq", "e2o");
 			const E2O_PATH_EXISTS = fs.existsSync(E2O_PATH);
-			const USING_E2O = channelAdapter === "e2o";
-			if (USING_E2O && !E2O_PATH_EXISTS) {
-				throw "Cannot use e2o channelAdapter unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
+			const USING_Electron = container === "electron";
+			if (USING_Electron && !E2O_PATH_EXISTS) {
+				throw "Cannot use electron container unless e2o optional dependency is installed. Please run npm i @chartiq/e2o";
```

Comment:
Is the package name changing?

Reply:
At some unknown point in the very near future.